### PR TITLE
[ADD] attachment_preview

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

File: attachment_preview/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+#
+##############################################################################
+{
+    "name": "Preview attachments",
+    "version": "1.0",
+    "author": "Therp BV",
+    "license": "AGPL-3",
+    "complexity": "normal",
+    "description": """
+This addons allows to preview attachments supported by http://viewerjs.org.
```
Comment (pedrobaeza) [inline attachment_preview/__openerp__.py]: s/addons/addon. It can be also 'module'

File: attachment_preview/model/ir_attachment.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        for this in self.browse(
+                cr, uid,
+                ids if isinstance(ids, collections.Iterable) else [ids],
+                context=context):
+            extension = ''
+            if this.datas_fname:
+                filename, extension = os.path.splitext(this.datas_fname)
+                extension = extension.lstrip('.')
+            if not extension:
+                # TODO: lookup the extension via mimetype
```
Comment (pedrobaeza) [inline attachment_preview/model/ir_attachment.py]: I think a lot of attachments falls in this case, so it's very needed this part. https://docs.python.org/2/library/mimetypes.html can be used.

**Comment by hbrunn** [comment]
Thanks Pedro for you input!

**Comment by pedrobaeza** [comment]
:+1:

**Comment by bwrsandman** [comment]
:+1:

**Pull request merged**